src/utils/pqUtils.ts

```diff
@@ -69,7 +71,7 @@ const getCustomXmlFile = async (
             break;
```

Comment:
why break and not continue?

Reply:
Will do tha same, break will return { found: false } meaning that there are no more xml files to loop through, but since we now get the files number, it's the same.